packages/icons-react/src/__tests__/createFromInfo-test.js

```diff
@@ -125,6 +125,18 @@ describe('createFromInfo', () => {
       }
     });
 
+    fit('should forward refs to rendered DOM element', async () => {
+      const moduleSource = createModuleFromInfo(info);
+      const MockIconComponent = await getModuleFromString(moduleSource);
+      let svg;
+      const ref = jest.fn(node => {
+        svg = node;
+      });
+      ReactDOM.render(<MockIconComponent ref={ref} />, mountNode);
+      expect(() => <MockIconComponent ref={ref} />).not.toThrow();
```

Comment:
What are we testing won't throw in this situation?

Reply:
that was holdover test from when I was getting the test setup, I've removed it